### PR TITLE
Add AsyncAPI WIP

### PR DESCRIPTION
**Pull request by jamilbk**
Adds AsyncAPI for the gateway.

### CODE REVIEW

File: asyncapi.yml [hunk 1]
```diff
@@ -0,0 +1,237 @@
...
+                    - 100.64.11.22/32
+                  public_key: AxVaJsPC1FSrOM5RpEXg4umTKMxkHkgMy1fl7t1xNyw=
+                  preshared_key: LZBIpoLNCkIe56cPM+5pY/hP2pu7SGARvQZEThmuPYM=
+                  user_uuid: 3118158c-29cb-47d6-adbf-5edd15f1af17
+
+  schemas:
+    init:
+      summary: Gateway initialization
+      description: |
+        This message indiciates the gateway should reconfigure itself with the contained configuration.
```
Comment (conectado) [inline asyncapi.yml:119]: Will this be the init only for the gateway and not for the client?

**Review by conectado — APPROVED** [review]
LGTM

For the gateway we still need to add a method to push its public key and the interface stats.

**Comment by jamilbk** [comment]
> LGTM
> 
> For the gateway we still need to add a method to push its public key and the interface stats.

File: asyncapi.yml (hunk 1 above)
Comment (jamilbk) [reply asyncapi.yml:119]: Yes. Client api will be synchronous and documented with openapi

**Pull request merged**